cmake/vcpkg-configuration.json

```diff
@@ -0,0 +1,25 @@
+{
+    "default-registry": {
+        "kind": "git",
+        "repository": "https://github.com/Microsoft/vcpkg",
+        "baseline": "1de2026f28ead93ff1773e6e680387643e914ea1"
+    },
+    "registries": [
+        {
+            "repository": "https://github.com/microsoft/vcpkg",
+            "kind": "git",
+            "packages": [
+                "flatbuffers"
+            ],
+            "baseline": "8161f655436492669adc8eb6f269dc41392ffa70"
+        },
+        {
+            "repository": "https://github.com/luncliff/vcpkg-registry",
```

Comment:
This is your personal repo?

Reply:
Yes. Using onyx 1.16.0 from vcpkg upstream is contained in https://github.com/microsoft/vcpkg/pull/36850
The part can be removed after the PR